src/cheri/riscv-hybrid-integration.adoc

```diff
@@ -208,21 +208,28 @@ In {cheri_int_mode_name}, the capability authorizing the memory access is <<ddc>
+//{cheri_base_ext_name} extends all RISC-V CSRs which hold addresses, to hold capabilities.
+//Therefore, such registers are extended to CLEN bits in {cheri_base_ext_name} and a `c` suffix is added when referring to the entire CSR instead of the only the address field.
 
-WARNING: This assumes that the Zcmt CSR is extended to be a capability when the base architecture is {cheri_base_ext_name}.
+//WARNING: This assumes that the Zcmt CSR is extended to be a capability when the base architecture is {cheri_base_ext_name}.
 
-For example, the `jvt` CSR is extended to `jvtc`, the `c` suffix indicating that it's a capability.
-It is referred to as `jvt` if only accessing the address or `jvtc` if accessing the whole capability.
+//For example, the `jvt` CSR is extended to `jvtc`, the `c` suffix indicating that it's a capability.
+//It is referred to as `jvt` if only accessing the address or `jvtc` if accessing the whole capability.
 
-* In {cheri_cap_mode_name} <<CSRRW_CHERI>> _always_ read/writes the entire CLEN-bits of all CLEN-width CSRs, i.e., it accesses `jvtc`.
-* In {cheri_int_mode_name} <<CSRRW_CHERI>> _only_ accesses XLEN-bits of extended CSRs (e.g., `jvt`, for compatibility) and all CLEN-bits of CSRs added by CHERI extensions (e.g., <<ddc>>).
-** <<CSRRW_CHERI>> in {cheri_int_mode_name} can only update the address field of `jvt`, but it can write a full value into <<ddc>>.
-** {cheri_cap_mode_name} always accesses `jvtc`, never `jvt`.
-* when only writing XLEN-bits to an extended CSR (e.g., `jvt`), the CSR value is updated using <<SCADDR>> semantics.
+When in {cheri_int_mode_name}, there is a special rule for updating extended CSRs (e.g., <<utid>>/<<utidc>> or <<jvt>>/<<jvtc>>)
 
-IMPORTANT: need to resolve how to cross-reference the privileged spec to have a list of the new and extended CSRs
+* Writing an extended CSR writes the address field (XLEN-bits) only, and the full CSR is updated using <<SCADDR>> semantics.
+* Reading an extended CSR reads the address field (XLEN-bits) only
+
+For this purpose the assembly syntax will refer to <<jvt>> or <<utid>>.
+
+//* In {cheri_cap_mode_name} <<CSRRW_CHERI>> _always_ read/writes the entire CLEN bits of all CLEN-width CSRs, i.e., it accesses `jvtc`.
+//* In {cheri_int_mode_name} <<CSRRW_CHERI>> _only_ accesses XLEN bits of extended CSRs (e.g., `jvt`, for compatibility) and all CLEN-bits of CSRs added by CHERI extensions (e.g., <<ddc>>).
+//** <<CSRRW_CHERI>> in {cheri_int_mode_name} can only update the address field of `jvt`, but it can write a full value into <<ddc>>.
+//** {cheri_cap_mode_name} always accesses `jvtc`, never `jvt`.
+//* when only writing XLEN bits to an extended CSR (e.g., `jvt`), the CSR value is updated using <<SCADDR>> semantics.
+
+//IMPORTANT: need to resolve how to cross-reference the privileged spec to have a list of the new and extended CSRs
```

Comment:
It seems those lines got copied over, but are commented out. Do you want them to appear in the actual spec or not?